awx/ui_next/src/components/Schedule/ScheduleOccurrences/ScheduleOccurrences.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import { shape } from 'prop-types';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { formatDateString, formatDateStringUTC } from '@util/dates';
+import { Split, SplitItem, TextListItemVariants } from '@patternfly/react-core';
+import { DetailName, DetailValue } from '@components/DetailList';
+import MultiButtonToggle from '@components/MultiButtonToggle';
+
+function ScheduleOccurrences({ preview = { local: [], utc: [] }, i18n }) {
+  const [mode, setMode] = useState('local');
+
+  if (preview.local.length < 2) {
+    return null;
+  }
+
+  return (
+    <>
+      <DetailName
+        component={TextListItemVariants.dt}
+        fullWidth
+        css="grid-column: 1 / -1"
+      >
+        <Split gutter="sm">
+          <SplitItem>
+            <div className="pf-c-form__label">
```

Comment:
Could we define this stuff at the top of the file as styled components or w imported PF components (instead of using this className calls/css rule.

Reply:
👍